src/App.vue

```diff
@@ -78,6 +84,25 @@ export default {
      */
     toggleFavourites() {
       this.showFavourites = !this.showFavourites;
+    },
+    /**
+     * Updates myFavourites array of beer ID's.
+     */
+    updateMyFavourites(id) {
+      const beerId = !!this.myFavourites.find(beer => beer === id);
```

Comment:
`beerId` is a bit confusing as this is not was is actually sending back the id. `isBeerFavourite`, as you are saving a boolean is better.

Reply:
Yes it was 😃